### PR TITLE
Update client repo to a new structure

### PR DESCRIPTION
**Pull request by victoriari**

### CODE REVIEW

File: client/src/app/components/page-layout/page-layout.tsx [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+export interface PageLayoutProps {
+	children?: React.ReactNode;
+	title?: string;
+	renderHeader?: () => React.ReactNode;
+}
+
+export const PageLayout = (props: PageLayoutProps) => {
+	const { children, renderHeader, title } = props;
+
+	useEffect(() => {
```
Comment (eugenebelov) [inline client/src/app/components/page-layout/page-layout.tsx:33]: not sure if we need a useEffect here :)

File: client/src/app/scenes/home/index.tsx [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+
+const HomeScene = () => {
+	const [showModal, setShowModal] = useState<boolean>(false);
+	return (
+		<>
+			<JackpotInfo setModalOpen={setShowModal} />
+			<StyledPageLayout title={'Casper Lottery'}>
+				<Plays setModalOpen={setShowModal} />
+				<Modal isOpen={showModal} setModalOpen={setShowModal} />
+			</StyledPageLayout>
```
Comment (eugenebelov) [inline client/src/app/scenes/home/index.tsx:20]: Here is a reason to use `useCallback` :) 
If we need to pass callback from state, need to wrap it in `useCallback`

But, I'm not sure if we need to pass `setShowModal` inside each of this component and manage state there. Probably better to do smth like that:

```
// here we can pass optional param if necessary
const handleGameStart = ( parameter ) => {
 setShowModal(true);
}
...
<JackpotInfo onGameStart={handleGameStart} />
```

File: client/src/app/services/hooks/use-manage-play.ts [hunk 3]
```diff
@@ -0,0 +1,154 @@
...
+			console.error(`Deploy failed: ${deploy.error_message}`);
+			setPlayResultState({
+				...playResultState,
+				data: DeployFailed.Failed,
+				loading: false,
+			});
+		}
+	};
+
+	useEffect(() => {
```
Comment (eugenebelov) [inline client/src/app/services/hooks/use-manage-play.ts:136]: move this effect at top. Just to better readability

File: client/src/app/scenes/play-modal/ModalState.tsx [hunk 4]
```diff
@@ -0,0 +1,142 @@
...
+	closeModal: () => void;
+}
+
+const ModalState = (props: ModalStateProps) => {
+	const { closeModal } = props;
+
+	const {
+		data: playResult,
+		loading: awaitingPlayResult,
+		error: clientErrorOccurred,
```
Comment (eugenebelov) [inline client/src/app/scenes/play-modal/ModalState.tsx:76]: let's simplify that to `loadingGameResult` and `errorGame` or smth like that

**Review by eugenebelov — COMMENTED** [review]
general stuff - I think prettier is broken :) just copy it from cspr.live, should helps.

**Review by eugenebelov — APPROVED** [review]

**Comment by eugenebelov** [comment]
@victoriari ignore my comment with `prettier` i think it's a difference between our IDE's. 
Overall - you did a great job! Well done! 👏💪

**Review by Volodymyr-Kuchinskyi — APPROVED** [review]
Configuration part LGTM

**Pull request merged**